### PR TITLE
rename osmtogtfs to o2g

### PR DESCRIPTION
**Pull request by mehdisadeghi**
@carocad please see the changes. This makes hispofer/osmtogtfs docker hub and osmtogtfs package on pypi obsolete. Instead, there are corresponding new entries named o2g.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ deploy:
     tags: true
     python: 3.6
     branch: master
```
Comment (carocad) [inline .travis.yml]: is this a contradiction? is says `branch: master` and right below it says `    condition: $TRAVIS_BRANCH =~ ^master$`, which I guess mean `NOT master`

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,540 @@
...
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+## ODC Open Database License (ODbL)
```
Comment (carocad) [inline LICENSE]: correct me if I am wrong but I thought that ODBL is only for databases not for source code 🤔

File: .travis.yml (hunk 1 above)
Comment (mehdisadeghi) [reply .travis.yml]: I know it is confusing. I had a tag published from PR branch and after reading travis docs about `on` I realized with `tags` entry travis ignores branch name. Apparently they consider tags and branch names mutually exclusive. So until I can update my understanding about the best release approach I decided to add the above condition to make sure nothing will be released from a pr branch.

Comment (mehdisadeghi) [reply .travis.yml]: I know it is confusing. I had a tag published from PR branch and after reading travis docs about `on` I realized with `tags` entry travis ignores branch name. Apparently they consider tags and branch names mutually exclusive. So until I can update my understanding about the best release approach I decided to add the above condition to make sure nothing will be released from a pr branch.

Comment (mehdisadeghi) [reply .travis.yml]: I know it is confusing. I had a tag published from PR branch and after reading travis docs about `on` I realized with `tags` entry travis ignores branch name. Apparently they consider tags and branch names mutually exclusive. So until I can update my understanding about the best release approach I decided to add the above condition to make sure nothing will be released from a pr branch.

Comment (mehdisadeghi) [reply .travis.yml]: Please see: https://docs.travis-ci.com/user/deployment#conditional-releases-with-on

**Pull request merged**